src/components/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+import { Filter } from './Filter';
+
+type Props = {
+  onFilter: (value: Status) => void;
+  onUncompletedTodos: () => Todo[];
+  currentFilterStatus: Status;
+  todos: Todo[];
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onUncompletedTodos,
+  onFilter,
+  currentFilterStatus,
+}) => {
+  return (
+    <>
+      {!!todos.length && (
```

Comment:
Move this check to the parent component, you don't need to render `Footer` in this case